Gandalf/build.sh

```diff
@@ -1,119 +1,98 @@
 #!/bin/bash
+# 
+# Packages "Gandalf" into a deb
+#
+# Follow https://google.github.io/styleguide/shell.xml
```

Comment:
As I saw it for the first time I though something like "What's this for? Maybe how to run a shell script? No it has to do something with styling...)